lib/generators/solidus_stripe/install/templates/config/initializers/solidus_stripe.rb

```diff
@@ -11,6 +11,10 @@
   # Number of seconds while a webhook event is valid after its creation.
   # Defaults to the same value as Stripe's default.
   # config.webhook_signature_tolerance = 150
+  #
+  # Name of the `Spree::RefundReason` used for Stripe-generated refunds.
+  # Defaults to {SolidusStripe::DEFAULT_STRIPE_REFUND_REASON_NAME}.
+  # config.default_refund_reason_name = "Refund generated from Stripe"
```

Comment:
Shouldn't it be `refund_reason_name`? The default is this one: SolidusStripe::DEFAULT_STRIPE_REFUND_REASON_NAME, and when someone changes the preference value, they are no more using the default.

Reply:
:100: changed